docs/design/design-reviews/trade-studies/template.md

```diff
@@ -36,6 +36,8 @@ The following section should establish the desired capabilities of the solution
 
 > **IMPORTANT** This is **not** intended to define outcomes for the design activity itself. It is intended to define the outcomes for the solution being designed.
 
+As mentioned in the [User Interface](./../../../user-interface-engineering/README.md) section, if the trade study is analyzing an application development solution, make use of the _persona stories_ to derive desired outcomes. For example, if a persona story exemplifies a certain accessibility requirement, the parrallel desired outcome may be "The application must be accessible for people with vision-based disabilities".
```

Comment:
> nit: I prefer sentences on newlines in Markdown

I don't think this is being enforced throughout the repo anymore so I'm going to leave as is so that this PR doesn't introduce too many changes in the diff.